source/index.d.ts

```diff
@@ -1,17 +1,20 @@
 import { Browser, BrowserFetcher, ChromeArgOptions, ConnectOptions, FetcherOptions, LaunchOptions } from 'puppeteer';
```

Comment:
@G-Rath I guess this is no longer needed?

Reply:
Yeah, weird wonder why my IDE didn't flag it 🤔